setup.py

```diff
@@ -91,6 +92,11 @@ def run(self):
     install_requires=REQUIRED,
     test_suite="tests",
     tests_require=TEST_REQUIRED,
+    extras_require={
```

Comment:
is this still needed?

Reply:
Yes, I thought that it would allow users to install winrt backend at the same time as bleak if they desired:

```
pip install bleak[winrt]
```
This will play nicely with the "if winwrt is installed, use that", will it not?

Reply:
Yes, that makes sense.

Reply:
For that matter, we could also add `pythonnet` here since it is possible to install it from source in Python 3.9.

Reply:
👍🏻